language/move-prover/src/prelude.bpl

```diff
@@ -1031,21 +1031,25 @@ procedure {:inline 1} Signature_ed25519_threshold_verify(bitmap: Value, signatur
 // Serialize is modeled as an uninterpreted function, with an additional
 // axiom to say it's an injection.
 
-function $LCS_serialize(ta: TypeValue, v: Value): Value;
+function {:inline} $LCS_serialize($m: Memory, ta: TypeValue, v: Value): Value {
+    $LCS_serialize_core(ta, v)
+}
+
+function $LCS_serialize_core(ta: TypeValue, v: Value): Value;
```

Comment:
What would be the reason to differentiate `$LCS_serialize` ("public" one) from `$LCS_serialize_core` ("private one")?

Reply:
This situation appears in other places including `$Hash_sha3`. Should we make yours as a convention?

Reply:
I tried to do here before in the axiomatization `forall $m: Memory :: ..`, but I got tests failing. So my conjecture was that the outer quantifier was not eliminated. I then used the trick above to avoid quantifying over memory, and it worked. 

Whether we need to do this at other places, we see. All native spec functions need to get the `$m` parameter added, but it appeared to me this was the only one. Defined spec functions are correct by compilation.

Reply:
I see. Actually, there are a couple of more native spec functions for Hash in the upstream, which landed some hours ago.

Reply:
Oops. Sorry for that, perhaps I submitted too fast. Let's fix it soon, given that this will show up in MIRAI bot and will bother now also folks working on move-lang, every time.